src/mscorlib/src/Interop/Windows/kernel32/Interop.Globalization.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
These were probably created by my automated process when it rewrote the files. I'll see if i can clean them up.